### ISSUE TITLE
The list of banned users in admin is too long

### ISSUE THREAD

**Issue by skariko** [LemmyNet/lemmy-ui#1821]
### Requirements

- [X] This is a bug report, and if not, please post to https://lemmy.ml/c/lemmy_support instead.
- [X] Please [check](https://github.com/LemmyNet/lemmy-ui/issues) to see if this issue already exists.
- [X] It's a single bug. Do not report multiple bugs in one issue.
- [X] It's a frontend issue, not a backend issue; Otherwise please create an issue on the [backend repo](https://github.com/LemmyNet/lemmy) instead.

### Summary

When you open the /admin page for some configuration the list of banned users immediately appears on the right. But it doesn't seem to me to be those banned by administrators (we may have banned one or two) but are they perhaps the users banned by anyone within the instance?

The thing is, the list is very long and keeps getting longer and longer and it also takes several minutes to load: that's not what I want the moment I click on /admin.

A quick suggestion for improvement in my opinion is to simply put it in a separate tab and have it load only at the time it is of interest.

### Steps to Reproduce

1. Go to youristance.xxx/admin
2. See the amount of banned users

### Technical Details

On mobile and desktop

### Lemmy Instance Version

0.18.0

**Comment by randommouse** [comment]
The number of banned users is unavoidable because of the nature of federation. I agree that it would be nice to have this list in a separate tab. I also think that the federated list should be separate from the list of bans that were initiated locally.

**Comment by slash-zero** [comment]
Was considering submitting a bug for this issue as well.  A short-term solution might be to put the list of banned users into a combo box or a text area, something with a fixed height and its own scroll bar.  That way it can't eventually cause a scroll bar for 100s of pages. 

A long-term solution would be to put it in a dedicated tab with a nice sortable table with pagination, search ability, etc.

**Comment by skariko** [comment]
> The number of banned users is unavoidable because of the nature of federation. I agree that it would be nice to have this list in a separate tab. I also think that the federated list should be separate from the list of bans that were initiated locally.

Exactly, the problem is not the list itself because it is correct that it is there. The problem is that it appears as soon as you press on /admin and on my server it also takes up to 2 minutes to load the whole page.

**Comment by Die4Ever** [comment]
I would like to point out that this issue is still part of the 0.18.2 milestone, which is hidden because that milestone is closed even though this issue is not. And there are other open issues which are hidden in the same way.

https://github.com/LemmyNet/lemmy-ui/milestones?state=closed

**Comment by Nutomic** [comment]
This was changed to show only local users, and moved to a separate tab.
